### DISCUSSION TITLE
Cmp support?

### DISCUSSION THREAD

**Discussion by TeoDev1611** [nvim-neorg/neorg#87] [Q&A] [answered]
Hi 👋 
Thanks for this amazing plugin ( is the most useful for me )... Currently have support for Cmp I only found Compe support.
If exists Cmp support can you add into the README or if not exists can make a issue 🙂

**Comment by vhyrro** [comment, marked as answer]
Hey man, you can check out the unstable branch's README for instructions on how to do that. cmp support is only currently available on that branch. If you do switch be sure to update your nvim-treesitter snippet with the one present there :)

**Reply by TeoDev1611** [reply]
Thanks
